src/OmniSharp/Settings/FromBodyApplicationModelConvention.cs

```diff
@@ -15,8 +15,8 @@ public void Apply(ApplicationModel application)
                     foreach (var parameter in action.Parameters)
                     {
                         if (parameter.BindingInfo?.BindingSource != null ||
-                            parameter.Attributes.OfType<IBindingSourceMetadata>().Any() ||
-                            ValueProviderResult.CanConvertFromString(parameter.ParameterInfo.ParameterType))
+                            parameter.Attributes.OfType<IBindingSourceMetadata>().Any())
+                            //|| ValueProviderResult.CanConvertFromString(parameter.ParameterInfo.ParameterType))
```

Comment:
yeah technically it's not needed because omnisharp doesn't use anything bound from route or querystring atm. 
But in the future if anyone would introduce something like this for example (for whatever reason), it would be failing:

```csharp
       [HttpPost("v{version:int}/findusages")]
       public async Task<QuickFixResponse> FindUsages(int version, FindUsagesRequest request)
```
But of course there's YOLO approach and you can just remove it for now and it will be brought back only if it's ever needed :)